config/config.go

```diff
@@ -648,6 +649,7 @@ func SetupViper(v *viper.Viper, filename string) {
 	v.SetDefault("adapters.rtbhouse.endpoint", "http://prebidserver-s2s-ams.creativecdn.com/bidder/prebidserver/bids")
 	v.SetDefault("adapters.somoaudience.endpoint", "http://publisher-east.mobileadtrading.com/rtb/bid")
 	v.SetDefault("adapters.sovrn.endpoint", "http://ap.lijit.com/rtb/bid?src=prebid_server")
+	v.SetDefault("adapters.synacormedia.endpoint", "http://{{.Host}}.technoratimedia.com/openrtb/bids/{{.Host}}")
```

Comment:
Do we expect `.technoratimedia.com` after the host value parses? Or did we mean `http://{{.Host}}/openrtb/bids/{{.Host}}`?

Reply:
The .technoratimedia.com is correct

Reply:
ok